var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -345,6 +347,13 @@ class Llvm(CMakePackage):
     # Github issue #4986
     patch('llvm_gcc7.patch', when='@4.0.0:4.0.1+lldb %gcc@7.0:')
 
+    patch('https://github.com/llvm-mirror/clang/pull/33.diff',
+        when='@4.0:4.0.9999 +flang',
+        sha256='e46d7ab305e5e95c51f4656d9b52058143cd85d859b312b3c80e93a02d54b4a5',
+        level=1, working_dir='tools/clang')
```

Comment:
Since this is a patch to support a downstream feature, should it be moved to the `flang` package?  See #5476 or the [new docs](https://spack.readthedocs.io/en/latest/packaging_guide.html#dependency-dependency-patching) for how to do that.  Doing this also eliminates the need for the extra conflicts and declutters the LLVM package.